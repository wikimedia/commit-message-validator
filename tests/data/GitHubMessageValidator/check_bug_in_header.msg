Fix bug #12344

This test should fail since bug is defined
