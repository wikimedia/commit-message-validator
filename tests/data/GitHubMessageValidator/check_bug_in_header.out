commit-message-validator
Using GitHubMessageValidator to check the commit message
The following errors were found:
Line 1: Do not define bug in the subject
